spec/lib/miq_automation_engine/methods/wait_for_ip_spec.rb

```diff
@@ -0,0 +1,34 @@
+require "spec_helper"
+require Rails.root.join('db/fixtures/ae_datastore/ManageIQ/ConfigurationManagement/AnsibleTower/Operations/StateMachines/Job.class/__methods__/wait_for_ip').to_s
+require Rails.root.join('spec/support/miq_ae_mock_service').to_s
```

Comment:
This feels like the wrong way to test this and is probably why your tests are failing with an exit code.

If you want to test the method it's probably better to create a spec in `spec/automation/unit/method_validation`